Network/Units/Client/Client.cpp

```diff
@@ -99,7 +95,11 @@ void Client::_startReceive()
 void Client::sendMessage(std::vector<uint8_t> message, uint32_t token)
 {
     (void) token;
-    _udp_socket->send_to(asio::buffer(message), *_server_endpoint);
+    try {
+        _udp_socket->async_send_to(asio::buffer(message), *_server_endpoint, [](const asio::error_code&, std::size_t) {});
+    } catch (const std::exception& e) {
+        std::cout << "Error in send Message client: " << e.what() << std::endl;
+    }
```

Comment:
```suggestion
    try {
        _udp_socket->send_to(asio::buffer(message), *_server_endpoint);
    } catch (const std::exception& e) {
        std::cout << "Error in send Message client: " << e.what() << std::endl;
    }
```
This is very important to keep that, because on Windows it crash.